src/lightning_app/components/serve/python_server.py

```diff
@@ -70,8 +70,15 @@ def request_code_sample(url: str) -> str:
 response = requests.post('"""
             + url
             + """', json={
-    "image": img
-})"""
+"image": img
+})
+# If you are using basic authentication for your app, you should add your credentials to the request:
+# response = requests.post('"""
+            + url
+            + """', json={
+# "image": img
+# }, auth=requests.auth.HTTPBasicAuth('your_username', 'your_password'))
```

Comment:
the comment marker for these lines is probably not intentional?

Reply:
It's intentional - just to (1) keep both options (creds-less and with credentials) at the same example and (2) keep the previous option (without credentials) as a default one.

Reply:
That's very unfortunate. If the user copies the code here (which is I assume why we show it), they get a syntax error due to missing braces. 
Also, the code formatting would look much more readable if we used `textwrap.dedent`.